test-a11y/config.js

```diff
@@ -1,8 +1,5 @@
 /* eslint global-require: 0, import/no-dynamic-require: 0 */
```

Comment:
I think we can probably remove these exceptions now, since we're no longer requiring external files?